x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -192,6 +192,11 @@ export function transformSavedObjectToExternalModel(
   const status =
     STATUS_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.status] ?? CaseStatuses.open;
 
+  // @ts-ignore
+  delete caseSavedObject.attributes?.total_alerts;
```

Comment:
Let's not mutate the `caseSavedObject` and spread to remove the fields as @jonathan-buttner suggested.